partners/anaconda/base/anaconda-ce-validator-cron.yaml

```
@@ -24,7 +24,7 @@ spec:
             - name: addon-managed-odh-pullsecret
           containers:
           - name: anaconda-ce-validator
-            image: quay.io/openshift/origin-deployer:4.7.0
+            image: $(image-sha-256-origin-deployer)
```

We are re-using the deployer image for anaconda job but it could be any other image:

```suggestion
            image: $(odh-anaconda-ce-validator-image)
```